great_expectations/checkpoint/configurator.py

```diff
@@ -51,7 +59,7 @@ def __init__(
         self,
         name: str,
         data_context,
-        site_names: Union[str, List[str]] = "all",
+        site_names: Union[None, str, List[str]] = "all",
         slack_webhook: Optional[str] = None,
         notify_on: str = "all",
         notify_with: Union[str, List[str]] = "all",
```

Comment:
is `all` one of a few options? If so, let's use `Union[Literal[...], Literal[...]]` here

Reply:
@cdkini Could you please put this idea into a separate JIRA ticket for us to follow up on?  Thanks!